external/fv3fit/fv3fit/_shared/packer.py

```diff
@@ -205,14 +207,18 @@ def dump(self, f: TextIO):
                 "pack_names": self._pack_names,
                 "sample_dim_name": self._sample_dim_name,
                 "feature_index": multiindex_to_tuple(self._feature_index),
+                "packer_config": dataclasses.asdict(self._config)
+                if self._config is not None
+                else {},
             },
             f,
         )
 
     @classmethod
     def load(cls, f: TextIO):
         data = yaml.safe_load(f.read())
-        packer = cls(data["sample_dim_name"], data["pack_names"])
+        packer_config = dacite.from_dict(PackerConfig, data["packer_config"])
+        packer = cls(data["sample_dim_name"], data["pack_names"], packer_config)
```

Comment:
can this be done in a backwards compatible way? for old dense models that didn't have a packer config?